src/core/column/nth.h

```diff
@@ -33,7 +33,7 @@ class NTH_ColumnImpl : public Virtual_ColumnImpl {
     Groupby gby_;
 
   public:
-    NTH_ColumnImpl(Column&& col, int32_t n,  const Groupby& gby)
+    NTH_ColumnImpl (Column&& col, int32_t n,  const Groupby& gby)
```

Comment:
Space between the method name and a bracket is not really needed, because it is natural to write `f(x)` with no spaces. Just take a look at the rest of dt code.